testing/run_tests.py

```diff
@@ -283,89 +283,19 @@ def RunDartTest(build_dir, test_packages, dart_file, verbose_dart_snapshot, mult
 
 def EnsureDebugUnoptSkyPackagesAreBuilt():
   variant_out_dir = os.path.join(out_dir, 'host_debug_unopt')
-
-  ninja_command = [
-    'ninja',
-    '-C',
-    variant_out_dir,
-    'flutter/sky/packages'
-  ]
-
-  # Attempt running Ninja if the out directory exists.
-  # We don't want to blow away any custom GN args the caller may have already set.
-  if os.path.exists(variant_out_dir):
-    RunCmd(ninja_command, cwd=buildroot_dir)
-    return
-
-  gn_command = [
-    os.path.join(buildroot_dir, 'flutter', 'tools', 'gn'),
-    '--runtime-mode',
-    'debug',
-    '--unopt',
-    '--no-lto',
-  ]
-
-  RunCmd(gn_command, cwd=buildroot_dir)
-  RunCmd(ninja_command, cwd=buildroot_dir)
+  assert os.path.exists(variant_out_dir), "%s doesn't exist. Run GN to generate the directory first" % variant_out_dir
```

Comment:
Can we at least tell them what commands to run?

I'm worried this will break local workflows and developers won't know what commands/targets were being built and just not bother running the tests locally.

Reply:
Updated the logic in https://github.com/flutter/engine/pull/26822 because I couldn't find a way to update this PR generated from the UI.